### ISSUE TITLE
Can't find any reference in some java files.

### ISSUE THREAD

**Issue by Carlos-yyt** [CoatiSoftware/Sourcetrail#838]
![TIM截图20191210000750](https://user-images.githubusercontent.com/35983679/70456257-c058c780-1ae8-11ea-861b-f806697815d7.png)
![TIM截图20191210010218](https://user-images.githubusercontent.com/35983679/70456259-c0f15e00-1ae8-11ea-826c-def7bdb81699.png)

As the 2 pics have showed, I downloaded some source and put them into Sourcetrail to find 0 references. In fact, no class was distinguished.

I try others java source too. Some could be distinguish but others couldn't. I can't find the law.

I guess didn't use this tool in a correct way...but how?

**Comment by Carlos-yyt** [comment]
It seems relates to the place your sources put?

Java source A couldn’t be distinguish, but source B could.I pasted source A into “/src” folder of source B,and it works.

How, i couldn’t figure out the difference between two places...

**Comment by egraether** [comment]
No, the files are found, but the indexer doesn't do much with them. Your code contains non ASCII symbols, I suppose Chinese? What text encoding do these files use?

**Comment by mlangkabel** [comment]
I don't think that this is a text encoding issue, because then we would get errors as described in #783. @Carlos-yyt, what does your Sourcetrail log say?
